tests/Microsoft.Identity.Test.Integration.netfx/SeleniumTests/DeviceCodeFlowIntegrationTest.cs

```diff
@@ -119,7 +119,10 @@ private async Task AcquireTokenWithDeviceCodeFlowAsync(LabResponse labResponse,
 
         #region Azure AD Kerberos Feature Tests
         [RunOn(TargetFrameworks.NetCore)]
-        [Timeout(2 * 60 * 1000)] // 2 min timeout
+        [Timeout(2 * 60 * 1000)] // 2 min timeout
+#if ONEBRANCH_BUILD

```

Comment:
We should not ignore the entire Kerberos test. I believe only `TestCommon.ValidateNoKerberosTicketFromAuthenticationResult` is failing, as that depends on some Windows API

Reply:
actually, there are tests that use `TestCommon.ValidateNoKerberosTicketFromAuthenticationResult` and they are running well. (see image below) Seems like something else is happening. not sure what it is. 

![image](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/assets/30090357/f7f3d30d-46fa-402c-b1b0-7294d497977b)